crates/interpreter/src/instructions/macros.rs

```diff
@@ -29,12 +37,16 @@ macro_rules! gas {
     };
 }
 
+/// Records a `gas` refund.
+#[macro_export]
 macro_rules! refund {
     ($interp:expr, $gas:expr) => {
         $interp.gas.record_refund($gas)
     };
 }
 
+/// Same as [`gas!`], but with `gas` as an option.
+#[macro_export]
```

Comment:
This macro calls `gas!`, if we want to consume it externally without importing also `gas!` we might need to change the call to `$crate::gas!`. Same for `as_usize_or_fail!` and `as_usize_or_fail_ret!`